addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -6480,11 +6480,6 @@ const ImageHandlerOption = SnippetOptionWidget.extend({
         const editableEl = this.$target[0].closest(".o_editable");
         await loadImageInfo(img, this.getParent().options.wysiwyg._getRecordInfo(editableEl), attachmentSrc);
         this.imageData = weUtils.getImageData(img, this.imageShape);
-        if (!this.imageData.originalId) {
```

Comment:
Removed as a fetched image has an `originalSrc` but no `originalId`. Since this commit, an image does not have to have an `originalId` to be editable.